.env.example

```diff
@@ -0,0 +1,6 @@
+    AUTH_DOMAIN = ""
```

Comment:
remove the whitespace before the variable names

Reply:
Changes have been made. Kindly review.